File: src/main/javabackup/de/thatsich/intellie/common/module/tileentity/ITileEntity.java
package de.thatsich.intellie.common.module.tileentity;

import net.minecraftforge.common.config.Configuration;

/**
 @author thatsIch
 @since 22.03.2014. */
public interface ITileEntity
{
	void register( Configuration config );

	String getTileEntityID();
}

File: src/main/scala/de/thatsich/minecraft/intellie/common/module/tileentities/ITileEntity.scala
package de.thatsich.minecraft.intellie.common.module.tileentities

import net.minecraftforge.common.config.Configuration

/**
 *
 *
 * @author thatsIch
 * @since 07.04.2014.
 */
/**
  */
trait ITileEntity
{
	def register(config: Configuration)

	def getTileEntityID: String
}